packages/fxa-content-server/app/scripts/templates/sms_send.mustache

```diff
@@ -9,21 +9,28 @@
     </header>
     <section class="send-sms">
       <div class="graphic {{graphicId}}" role="img" aria-label="{{#t}}Connect another device{{/t}}"></div>
+
       <h1 data-id="prompt-header">
```

Comment:
Since we're no longer _always_ displaying an `h1`, we'll want to move it inside the conditionals so a blank `h1` isn't rendered.